src/cider/nrepl/middleware/util/instrument.clj

```diff
@@ -285,7 +286,7 @@
                                  {::extras (assoc line-col :coor i)}))
                       form)
         ;; Expand so we don't have to deal with macros.
-        (m/macroexpand-all ::original-form)
+        (r/macroexpand-all)
```

Comment:
Doesn't this get rid of the original-form metadata? Some functionality relies on it.